elixir/apps/domain/lib/domain/telemetry/google_cloud_metrics_reporter.ex

```diff
@@ -15,7 +15,7 @@ defmodule Domain.Telemetry.GoogleCloudMetricsReporter do
 
   # Maximum time in seconds to wait before flushing the buffer
   # in case it did not reach the @buffer_size limit within the flush interval
-  @flush_interval :timer.seconds(15)
+  @flush_interval :timer.seconds(30)
```

Comment:
I did a quick search on the error being returned so often.  I may be wrong on this, but from some quick reading it looks like we should be able to send user-defined metrics every 5 seconds.  That makes me wonder, are we seeing that error due to multiple instances of this module being started and reporting to the same "name" in quick succession?

Reply:
Good catch, I think you're probably right. I'll take another look and see if we only need to fire up one of these, similar to what we did for Sentry.